tests/core/test_save_restore.py

```diff
@@ -134,12 +134,13 @@ def test_EncDecCTCModelBPE(self):
         cn = EncDecCTCModelBPE.from_pretrained(model_name="stt_en_citrinet_256")
         self.__test_restore_elsewhere(model=cn, attr_for_eq_check=set(["decoder._feat_in", "decoder._num_classes"]))
 
-    @pytest.mark.with_downloads()
-    @pytest.mark.unit
-    def test_EncDecCTCModelBPE_v2(self):
-        # TODO: Switch to using named configs because here we don't really care about weights
-        cn = EncDecCTCModelBPE.from_pretrained(model_name="stt_en_conformer_ctc_small")
-        self.__test_restore_elsewhere(model=cn, attr_for_eq_check=set(["decoder._feat_in", "decoder._num_classes"]))
+    # disabled until new models are released on NGC
+    # @pytest.mark.with_downloads()
+    # @pytest.mark.unit
+    # def test_EncDecCTCModelBPE_v2(self):
+    #     # TODO: Switch to using named configs because here we don't really care about weights
+    #     cn = EncDecCTCModelBPE.from_pretrained(model_name="stt_en_conformer_ctc_small")
+    #     self.__test_restore_elsewhere(model=cn, attr_for_eq_check=set(["decoder._feat_in", "decoder._num_classes"]))
```

Comment:
Why comment it out?

Reply:
This model is going to get updated with new version. So I updated the version number here, but it is not uploaded on NGC yet. We would enable it when NGC is updated.

Reply:
Got it!